### PR TITLE
Fixes from Pycharn errors/warnings

### PR DESCRIPTION
**Pull request by rhfogh**
All fixes are from errors or warnings signalled by pycharm.
I did not run black, to make it easier to see what has changed.
There are a number of NBNB or FIXME where I was not sure what to do.
A couple of cases (including 'enqueue' are because of superclass/subclass function signature mismatch.

### CODE REVIEW

File: mxcubecore/Command/Tango.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ def continue_init(self, _):
         # self.init_poller.stop()
 
         if isinstance(self.polling, int):
-            self.raw_device = DeviceProxy(self.deviceName)
+            self.raw_device = DeviceProxy(self.device_name)
```
Comment (marcus-oscarsson) [inline mxcubecore/Command/Tango.py:207]: good :)

**Comment by marcus-oscarsson** [comment]
Looks, good :) Thanks !

There seems to be a conflict for some reason, it would be good to have a look at it. I'm also personally in favor of removing code completely instead of just commenting it (except for in rare occasions).

File: mxcubecore/HardwareObjects/QtGraphicsManager.py [hunk 2]
```diff
@@ -1401,6 +1401,7 @@ def show_all_items(self):
                 shape.show()
 
     def inc_used_for_collection(self, cpos):
+        # This function is never called - it probably should be
```
Comment (IvarsKarpics) [inline mxcubecore/HardwareObjects/QtGraphicsManager.py:1404]: It is called in queue_entry.py :

https://github.com/mxcube/mxcubecore/blob/1557f032a385ec1dc0dda10107d920cdf32461eb/mxcubecore/HardwareObjects/queue_entry.py#L735

File: mxcubecore/HardwareObjects/queue_entry.py [hunk 3]
```diff
@@ -732,6 +732,7 @@ def collect_dc(self, dc, list_item):
                     )
                     HWR.beamline.collect.set_helical(False)
                     HWR.beamline.collect.set_mesh(True)
+                    # inc_used_for_collection does nothing
```
Comment (IvarsKarpics) [inline mxcubecore/HardwareObjects/queue_entry.py:735]: Hmm, what do you mean by does nothing? It increases the number of how many times a point is used in collections.

File: mxcubecore/Command/Mockup.py [hunk 4]
```diff
@@ -18,14 +18,14 @@
 #  You should have received a copy of the GNU Lesser General Public License
 #  along with MXCuBE. If not, see <http://www.gnu.org/licenses/>.
 
-import time
-import logging
-import weakref
+# import time
```
[the remark this answers is not in the document]
Comment (rhfogh) [reply mxcubecore/Command/Mockup.py:21]: Agreed. Just some times I (inconsistently) left them in to make clearer what had changed.

File: mxcubecore/HardwareObjects/QtGraphicsManager.py (hunk 2 above)
Comment (rhfogh) [reply mxcubecore/HardwareObjects/QtGraphicsManager.py:1404]: Oops!. My apologies. I had missed that QtGraphicsManager was a sample-view.

There are still the problems with this function that the signature of the abstractmethod in the superclass is incompatible with the signatures in the subclasses, and that the function in SampleView does nothing, so the function only has an effect under Qt. Unless I got something else wrong, of course.

File: mxcubecore/Command/Sardana.py [hunk 5]
```diff
@@ -116,6 +116,7 @@ class SardanaObject(object):
 
     def object_listener(self, *args):
         ev = AttributeEvent(args)
+        # NBNB self.update not defined
```
Comment (marcus-oscarsson) [inline mxcubecore/Command/Sardana.py:119]: Not introduced in this PR

File: mxcubecore/HardwareObjects/ESRF/ESRFMultiCollect.py [hunk 6]
```diff
@@ -8,7 +8,8 @@
 from mxcubecore.utils.conversion import string_types
 from mxcubecore import HardwareRepository as HWR
 
-from ESRF.ESRFMetadataManagerClient import MXCuBEMetadataClient
+from mxcubecore.HardwareObjects.ESRF.ESRFMetadataManagerClient import MXCuBEMetadataClient
+# NBNB nicoproc is not found
```
Comment (marcus-oscarsson) [inline mxcubecore/HardwareObjects/ESRF/ESRFMultiCollect.py:12]: Yes, indeed forgot to add this file, I will do it

File: mxcubecore/HardwareObjects/ISPyBClient.py [hunk 7]
```diff
@@ -1073,7 +1073,7 @@ def update_data_collection(self, mx_collection, wait=False):
     def update_bl_sample(self, bl_sample):
         """
         Creates or stos a BLSample entry.
-
+        # NBNB update doc string
```
Comment (marcus-oscarsson) [inline mxcubecore/HardwareObjects/ISPyBClient.py:1076]: Again not changed in this PR

**Pull request merged**